backend/src/govsg/services/govsg-callback.service.ts

```diff
@@ -226,6 +283,42 @@ const parseTemplateMessageWebhook = async (
       void govsgMessage?.update(fieldOpts, whereOpts)
       void govsgMessageTransactional?.update(fieldOpts, whereOpts)
       void govsgOp?.update(fieldOpts, whereOpts)
+      // TODO: Add logging statement here for traceability
```

Comment:
Did you mean to handle logging in this PR?

Reply:
Tbh I got a bit tired on Friday. Could I clean this up in a subsequent PR? This is because I want to think about what meaningful things can be logged. This line was originally something less useful like "Checking govsgMessage".

Reply:
Yup no problem. I think above we already discussed another PR for cleaning up logging stuff, we can just revisit all these whenever we can.